src/tests/tribol_comp_geom.cpp

```diff
@@ -292,36 +292,321 @@ TEST_F( CompGeomTest, poly_area_centroid_2 )
-//registerMesh() face 1 y: 2, 0.0902704, 0
-//(x0,y0) and (x1,y1): (0.744548, 0.0902704), (0.75, 0).
-//(px,py): (0.735935, 0.136655)
-//SegmentBasis: phi is 1.51907 not between 0. and 1.
-//(x0,y0) and (x1,y1): (0.75, 0), (0.744548, 0.0902704).
-//(px,py): (0.735935, 0.136655)
-//SegmentBasis: phi is 1.51907 not between 0. and 1.
-
-   // TODO update to use these face coordinates from testing
-//   xy1[0] = 0.75;
-//   xy1[1] = 0. ;
-//   xy1[2] = 0.744548; 
-//   xy1[3] = 0.0902704;
-//
-//   xy2[0] = 0.744548;
-//   xy2[1] = 0.0902704;
-//   xy2[2] = 0.75;
-//   xy2[3] = 0.;
+   // this geometry should NOT be in contact
+   xy1[0] = 0.324552;
+   xy1[1] = 0.625596;
+   xy1[2] = 0.16206;
+   xy1[3] = 0.722646;
+
+   xy2[0] = 4.59227e-17;
+   xy2[1] = 0.752178;
+   xy2[2] = 0.161705;
+   xy2[3] = 0.72276;
+
+   RealT x1[numVerts];
+   RealT y1[numVerts];
+   RealT x2[numVerts];
+   RealT y2[numVerts];
+
+   for (int i=0; i<numVerts; ++i)
+   {
+      x1[i] = xy1[i*dim];
+      y1[i] = xy1[i*dim+1];
+      x2[i] = xy2[i*dim];
+      y2[i] = xy2[i*dim+1];
+   }
+
+   tribol::IndexT conn1[2] = {0,1};
+   tribol::IndexT conn2[2] = {0,1};
+
+   tribol::registerMesh( 0, 1, 2, &conn1[0], (int)(tribol::LINEAR_EDGE), &x1[0], &y1[0], nullptr, tribol::MemorySpace::Host );
+   tribol::registerMesh( 1, 1, 2, &conn2[0], (int)(tribol::LINEAR_EDGE), &x2[0], &y2[0], nullptr, tribol::MemorySpace::Host );
+
+   RealT fx1[2] = {0., 0.};
+   RealT fy1[2] = {0., 0.};
+   RealT fx2[2] = {0., 0.};
+   RealT fy2[2] = {0., 0.};
+
+   tribol::registerNodalResponse( 0, &fx1[0], &fy1[0], nullptr );
+   tribol::registerNodalResponse( 1, &fx2[0], &fy2[0], nullptr );
+
+   tribol::setKinematicConstantPenalty( 0, 1. );
+   tribol::setKinematicConstantPenalty( 1, 1. );
+
+   tribol::registerCouplingScheme( 0, 0, 1,
+                                   tribol::SURFACE_TO_SURFACE,
+                                   tribol::NO_CASE,
+                                   tribol::COMMON_PLANE,
+                                   tribol::FRICTIONLESS,
+                                   tribol::PENALTY,
+                                   tribol::BINNING_GRID,
+                                   tribol::ExecutionMode::Sequential );
+
+   tribol::setPenaltyOptions( 0, tribol::KINEMATIC, tribol::KINEMATIC_CONSTANT );
+   tribol::setContactAreaFrac( 0, 1.e-4 );
+
+   RealT dt = 1.;
+   int update_err = tribol::update( 1, 1., dt );
+
+   EXPECT_EQ( update_err, 0 );
+
+   tribol::CouplingSchemeManager& couplingSchemeManager = 
+         tribol::CouplingSchemeManager::getInstance();
+
+   tribol::CouplingScheme* couplingScheme = 
+      &couplingSchemeManager.at( 0 );
+
+   EXPECT_EQ( 0, couplingScheme->getNumActivePairs() );
+
+}
+
+TEST_F( CompGeomTest, coincident_vertices_full_overlap )
+{
+   constexpr int dim = 2;
+   constexpr int numVerts = 2;
+   RealT xy1[dim*numVerts];
+   RealT xy2[dim*numVerts];
+
+   // this geometry should NOT be in contact
```

Comment:
Can you give a brief explanation about why tribol shouldn't think these edges aren't in contact?  Maybe something like "edges are too far away so xxx check will fail."